### PR TITLE
Add support for XLSX report output

### PR DESCRIPTION
**Pull request by cgdobre**
Added support for XLSX output for reports

### CODE REVIEW

**Comment by kriskbx** [comment]
Wow! Awesome job, this is really great. Thank you so much for this contribution! <3
I found a tiny bug in your code, can you fix that? I will merge your PR immediately afterwards. :)

File: src/output/xlsx.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+        if (this.config.get('report').includes('merge_requests')) {
+            XLSX.utils.book_append_sheet(workbook, this.xlsxMergeRequests, 'Merge Requests');
+        }
+
+        if (this.config.get('report').includes('records')) {
+            XLSX.utils.book_append_sheet(workbook, this.xlsxRecords, 'Records');
+        }
+
+        XLSX.writeFile(workbook, fileName + extName);
```
Comment (kriskbx) [inline src/output/xlsx.js]: `fileName = path.basename()` already contains the extension, so you get a double extension when saving files. Can you remove `extName` completely?

**Comment by cgdobre** [comment]
Fixed. Thanks for pointing that out!

**Comment by kriskbx** [comment]
I just released version 1.7.26 which includes your addition. Thank you very much!

**Comment by cgdobre** [comment]
Awesome! thanks for the quick handling!

**Pull request merged**